src/main/java/racingcar/controller/CarController.java

```diff
@@ -3,36 +3,36 @@
 import java.util.List;
 
 import racingcar.domain.Car;
-import racingcar.repository.CarRepository;
-import racingcar.service.CarService;
-import racingcar.service.RandomNumberGenerator;
+import racingcar.domain.RacingGame;
+import racingcar.domain.RacingCars;
+import racingcar.utils.RandomNumberGenerator;
 import racingcar.view.InputView;
 import racingcar.view.OutputView;
 
 public class CarController {
-	private CarRepository carRepository = new CarRepository();
+	private RacingCars racingCars = new RacingCars();
 	int roundCount;
 
 	public void run() {
-		while (!getCarNames()) ;
-		while (!getRoundCount()) ;
-		move(roundCount);
+		while (!inputCarNames()) ;
+		while (!inputRoundCount()) ;
+		startRound(roundCount);
 	}
 
-	private boolean getCarNames() {
+	private boolean inputCarNames() {
 		try {
 			OutputView.printCarNameRequestMsg();
 			List<String> carNames = InputView.readCarNames();
-			carNames.stream().forEach(carName -> carRepository.add(new Car(carName)));
+			carNames.stream().forEach(carName -> racingCars.add(new Car(carName)));
```

Comment:
```suggestion
			carNames.stream()
			    .forEach(carName -> racingCars.add(new Car(carName)));
```
스트림은 연산자마다 줄바꿈을 해줘야 가독성이 좋습니다 😉